Source/WebKit/WebProcess/Network/WebResourceLoader.cpp

```diff
@@ -98,15 +98,15 @@ void WebResourceLoader::detachFromCoreLoader()
 
 MainFrameMainResource WebResourceLoader::mainFrameMainResource() const
 {
-    RefPtr frame = m_coreLoader->frame();
+    RefPtr frame = protectedCoreLoader()->frame();
```

Comment:
Oh, in this function, we probably want to declare a local variable instead of repeatedly calling protectedCoreLoader():
```
RefPtr coreLoader = m_coreLoader;
```